include/grpc/grpc_security.h

```diff
@@ -881,6 +882,7 @@ struct grpc_tls_server_authorization_check_arg {
   const char* peer_cert;
   grpc_status_code status;
   const char* error_details;
+  grpc_tls_server_authorization_check_config* config;
```

Comment:
Similar to my above comment:

- can you add the new parameter "config" in the comment above to describe what it does?
- can you split the comment to describe each parameter on a new line?
- fix the discrepancy between "peer_cert" and "certificate" in the comment

Reply:
The newest commit changes the comments, per the above. Thanks for your comments Sanjay! :)